content/en/blog/falco-week-48/index.md

```diff
@@ -2,8 +2,10 @@
 title: Falco Weekly 48
 date: 2023-12-01
 author: Aldo Lacuku, Federico Di Pierro
-slug: falco-w-48
+slug: falco-w-48, 2023, weekly-recap
```

Comment:
```suggestion
slug: falco-w-48-2023-weekly-recap
aliases:
- falco-w-48
```

Reply:
This one should work! Apparently comma or dash separated slugs are the same, I'm fine with either

Reply:
Done thanks!